modules/recommended-requirements-central-services.adoc

```diff
@@ -52,6 +53,38 @@ The following table lists the minimum memory and CPU values required to run Cent
 | 16 GiB
 |===
 
+[id="recommended-requirements-central-db-services-central_{context}"]
+== Central-db
+
+[discrete]
+== Memory and CPU requirements
+
+The following table lists the minimum memory and CPU values required to run Central-db for one secured cluster. The table includes the number of concurrent web portal users.
+
+|===
+| Deployments | Concurrent web portal users | CPU | Memory
```

Comment:
I wasn't able to find docs where we have defined listed values. Can you please provide more info on where we have defined these values?

We have talked about the following values:

Deployments | Concurrent users | CPU | Memory
--- | --- | --- | ---
< 25k | 1 | 8 | 16 GB
< 25k | <5 | 16 | 16 GB
< 50k | 1 | 8 | 24 GB (or 32 GB)
< 50k | <5 | 16 | 24 GB (or 32 GB)

Reply:
This is the document where I did the analysis https://docs.google.com/document/d/1YbIHVLnwLGvQFc35wKPMHwBbhbZFI-SKG9rmkXqisJM/edit?tab=t.0#heading=h.kdzxu1msgans

This can certainly be improved.